examples/counter/.babelrc

```diff
@@ -1,13 +1,17 @@
 {
   "stage": 2,
-  "plugins" : [
-    "react-transform"
-  ],
-  "extra"  : {
-    "react-transform": [{
-      "target"  : "react-transform-webpack-hmr",
-      "imports" : ["react"],
-      "locals"  : ["module"]
-    }]
+  "env": {
+    "development": {
+      "plugins": [
+        "react-transform"
+      ],
+      "extra": {
+        "react-transform": [{
+          "target":  "react-transform-hmr",
+          "imports": ["react"],
+          "locals":  ["module"]
+        }]
+      }
+    }
   }
```

Comment:
`"react-transform"` needs to go into webpack config as a loader query. As of `react-transform-hmr@1.0.0` this will throw during server side rendering.